commit aa9022d182766905959583c22617c5ac6179d239 (refs/changes/81/2981/3)
Author: Marsell Kukuljevic <marsell@joyent.com>
Date:   2017-11-22T10:28:06+00:00 (1 year, 11 months ago)
    
    PUBAPI-1461: vuln in semver module in sdc-cloudapi
    Reviewed by: Pedro P. Candel <pedro@joyent.com>

diff --git a/package.json b/package.json
index 9dde7b0..bb23cae 100644
--- a/package.json
+++ b/package.json
@@ -26,7 +26,7 @@
         "krill": "1.0.1",
         "sdc-clients": "10.4.0",
         "ufds": "1.3.0",
-        "semver": "2.2.1",
+        "semver": "5.4.1",
         "nodemailer": "0.3.29",
         "clone": "0.1.5",
         "assert-plus": "1.0.0",
